### DISCUSSION TITLE
Şehirler arası yol sorunu sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#285] [Zeka Sorusu]
Bir ülkede A, B,C ve D şehirleri vardır. A şehriyle B,C,D şehirleri arasındaki direkt yolların sayısı sırasıyla 2, 2,3 tür. B şehriyle C,D şehirleri arasındaki direkt yolların sayısı sırasıyla 5,1; C ile D arasındaki direkt yolların sayısı ise 3 ’tür. Buna göre A şehrinden D şehrine, bir şehirden en fazla bir kez geçerek kaç farklı şekilde gidilebilir?

**Comment by slos8888** [comment]
A şehrinden D şehrine bir şehirden en fazla bir kez geçerek 8 farklı şekilde gidilebilir.

**Reply by sonsuzus** [reply]
Açıklamalı çözüyoruz.

**Comment by begumasyaburus** [comment]
AD-3 farklı yol şekli
ABD-4 farklı yol şekli 
ACD-12 farklı yol şekli
3+4+12=19

**Comment by Murkrow067** [comment]
A-D: 3 yol
A-B-D: 2\*1 yol
A-C-D: 2\*3 yol
A-B-C-D: 2\*5\*3 yol
A-C-B-D: 2\*5\*1 yol

3+2+6+30+10= 51 farklı şekilde gidebilir.
